src/style.css

```diff
@@ -0,0 +1,138 @@
+* {
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: bold;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+input {
+  font-family: "Avenir", Arial, Verdana, Tahoma, sans-serif;
+}
+
+.Big-input {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+
+  border-radius: 4px;
+
+  font-size: 16px;
+
+  display: flex;
+  padding-left: 26px;
+  padding-top: 23px;
+  padding-bottom: 21px;
+  margin-top: 20px;
```

Comment:
you have a lot of common styles for big and small inputs. Move them under one selector